.sparkswap/config.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Sparkswap configuration for use inside the container. (Note the rpcCertPath is relative to being inside the container)
```

Comment:
Can you rename this file to something more obvious, like `cli-container-config`, and not in hidden directory.